services/minespace-web/src/styles/components/Button.scss

```diff
@@ -10,15 +11,15 @@
   // these and the hover rules are really only for default,
   // but due to precendence rules, kept at top level
   color: variables.$secondary-btn-color; // #medium-grey
-  border: 1px solid variables.$secondary-btn-color;
+  border: 2px solid variables.$secondary-btn-color;
 
   &:hover {
     color: variables.$secondary-btn-color-hover;
     border-color: variables.$secondary-btn-border-color-hover;
   }
 
   span {
-    font-size: theme.$font-size-sm;
+    font-size: 14px;
```

Comment:
The cancel button was right beside the "Terminate EoR" button and the text was different sizes. I was not having it.